<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraCalViewPreferencePage</class>
 <widget class="QWidget" name="CameraCalViewPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="m_PreviousCalibrationDirLabel">
       <property name="toolTip">
        <string>The minimum before camera calibration starts.</string>
       </property>
       <property name="text">
        <string>previous calibration directory:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="m_PreviousCalibrationDirLineEdit">
         <property name="toolTip">
          <string>Plain text file containing point id and 3D (x, y, z) coordinates, one point per line.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_PreviousCalibrationDirToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="m_3DModelLabel">
       <property name="toolTip">
        <string>Plain text file containing point id and 3D (x, y, z) coordinates, one point per line.</string>
       </property>
       <property name="text">
        <string>model file:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="m_3DModelLineEdit">
         <property name="toolTip">
          <string>Plain text file containing point id and 3D (x, y, z) coordinates, one point per line.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_3DModelToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="m_ScaleImageLabel">
       <property name="toolTip">
        <string>Scale the image, changing the number of pixels.</string>
       </property>
       <property name="text">
        <string>rescale image:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="m_ScaleImageInXSpinBox">
         <property name="toolTip">
          <string>Scale image in X direction.</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="m_ScaleImageInYSpinBox">
         <property name="toolTip">
          <string>Scale image in Y direction.</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="m_NumberViewsLabel">
       <property name="toolTip">
        <string>Required number of views before camera calibration starts.</string>
       </property>
       <property name="text">
        <string>number of views</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="m_NumberViewsSpinBox">
       <property name="toolTip">
        <string>The minimum before camera calibration starts.</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="m_FeaturesLabel">
       <property name="toolTip">
        <string>Choose the features you are trying to detect.</string>
       </property>
       <property name="text">
        <string>features to detect:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="m_FeaturesComboBox">
       <property name="toolTip">
        <string>Select type of features.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="m_GridSizeLabel">
       <property name="toolTip">
        <string>For chessboards, and grids of dots or rings, specify the number of dots in horizontal then vertical direction.</string>
       </property>
       <property name="text">
        <string>grid size:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="m_GridPointsInXSpinBox">
         <property name="toolTip">
          <string>Number of features in X direction.</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>14</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_ByLabel">
         <property name="text">
          <string>by</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_GridPointsInYSpinBox">
         <property name="toolTip">
          <string>Number of features in Y direction.</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="m_TagFamilyLabel">
       <property name="toolTip">
        <string>Aptil Tags tag family code</string>
       </property>
       <property name="text">
        <string>tag family:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="m_TagFamilyComboBox">
       <property name="toolTip">
        <string>April Tag family name.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="m_MinPointsLabel">
       <property name="toolTip">
        <string>Normally, chessboards require all points,
 but for things like April Tags, you get varying numbers per image.</string>
       </property>
       <property name="text">
        <string>minimum number of points:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="m_MinPointsSpinBox">
       <property name="toolTip">
        <string>Minimum number of points for April Tags.</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="m_ReferenceImageLabel">
       <property name="toolTip">
        <string>See Dutta 2009.</string>
       </property>
       <property name="text">
        <string>reference image:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="m_ReferenceImageLineEdit">
         <property name="toolTip">
          <string>Reference image for iterative calibration.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_ReferenceImagePushButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="m_ReferencePointsLabel">
       <property name="toolTip">
        <string>See Dutta 2009.</string>
       </property>
       <property name="text">
        <string>reference points:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="m_ReferencePointsLineEdit">
         <property name="toolTip">
          <string>Reference points (2D) for iterative calibration.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_ReferencePointsPushButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="m_TemplateImageLabel">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>template image:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="m_TemplateImageLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="m_TemplateImagePushButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="m_Do3DOptimisationLabel">
       <property name="toolTip">
        <string>For stereo, optimises 3D reconstruction error rather than reprojection error.</string>
       </property>
       <property name="text">
        <string>do 3D optimisation:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="m_Do3DOptimisationCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="m_IterativeLabel">
       <property name="toolTip">
        <string>See Dutta 2009 paper.</string>
       </property>
       <property name="text">
        <string>do iterative calibration:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="m_IterativeCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="m_ClusteringLabel">
       <property name="toolTip">
        <string>See Dutta 2009 paper.</string>
       </property>
       <property name="text">
        <string>do clustering on circle detection</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="m_ClusteringCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="m_HandEyeLabel">
       <property name="text">
        <string>preferred hand-eye method:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="m_HandEyeComboBox">
       <property name="toolTip">
        <string>This plugin calculates all available hand-eye methods, so this is the one to visualise.</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="m_ModelTransformLabel">
       <property name="toolTip">
        <string>Name of a 4x4 file to describe model to tracker transform.</string>
       </property>
       <property name="text">
        <string>model transform</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="m_ModelTransformLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="m_ModelTransformToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="m_ModelIsStationaryLabel">
       <property name="toolTip">
        <string>Name of a 4x4 file to describe model to tracker transform.</string>
       </property>
       <property name="text">
        <string>model is stationary</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QCheckBox" name="m_ModelIsStationaryCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="m_OutputDirLabel">
       <property name="toolTip">
        <string>The minimum before camera calibration starts.</string>
       </property>
       <property name="text">
        <string>output directory:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLineEdit" name="m_OutputDirLineEdit">
         <property name="toolTip">
          <string>Plain text file containing point id and 3D (x, y, z) coordinates, one point per line.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_OutputDirToolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
